pages/fr/a-propos.js

```diff
@@ -0,0 +1,34 @@
+import fs from "fs";
+import matter from "gray-matter";
+import PatternPage from "../../components/templates/PatternPage";
+import dictionary from '../../content/fr/dictionary.json';
+
+export default function About(props) {
+
+    return (
+        <PatternPage t={dictionary} pattern={props.pattern}/>
```

Comment:
Again the use of `PatternPage` for a non-pattern page type

Reply:
Same as above